label-stick-be/service-gateway/src/services/assignment/resolver.py

```diff
@@ -0,0 +1,22 @@
+from .dto import CreateAssignmentInputDTO, AssignmentResponseDTO, UpdateAssignmentInputDTO
+from ...utils.dto import ResponseDTO
+from ...utils.rest_api import call_api, HttpMethod
+from ...configs.base import MANAGER_SERVICE_URL
+import strawberry
+from fastapi.encoders import jsonable_encoder
+
+
+async def resolve_create_assignment(input: CreateAssignmentInputDTO) -> ResponseDTO[AssignmentResponseDTO]:
```

Comment:
mấy cái API này sử dụng http method thay vì dùng toàn bộ là POST. sửa lại để nó dùng chung đường dẫn nhưng khác method nhé